prow/plugins/plugins.go

```diff
@@ -144,7 +144,8 @@ type Heart struct {
 type Label struct {
 	// SigOrg is the organization that owns the
 	// special interest groups tagged in this repo
-	SigOrg string `json:"sig_org,omitempty"`
+	SigOrg                 string `json:"sig_org,omitempty"`
+	MilestoneMaintainersId string `json:"milestone_maintainers_id,omitempty"`
```

Comment:
`MilestoneMaintainersID int`: capitalizes the ID and removes the need for the strconv check later on.